src/components/governance/PageProposal.vue

```diff
@@ -10,21 +10,27 @@
             {{ status.badge }}
           </span>
           <div class="proposal-title__row">
-            <router-link
-              :to="`/proposals/` + getPrevProposalId"
+            <div class="read-more-link"
+              @click="
+                $router.push({
+                  name: 'Proposal',
+                  params: { proposalId: String(getPrevProposalId) }
+                })"
               :style="{ visibility: getPrevProposalId ? 'visible' : 'hidden' }"
-              class="read-more-link"
             >
               <i class="material-icons notranslate">chevron_left</i>
-            </router-link>
+            </div>
             <h2 class="proposal-title">{{ proposal.title }}</h2>
-            <router-link
-              :to="`/proposals/` + getNextProposalId"
+            <div class="read-more-link"
```

Comment:
why was this change necessary?

Reply:
Yes, you can also do:

```
<router-link class="read-more-link" :to="{name: 'Proposal',
                  params: { proposalId: String(getPrevProposalId) }}" 
            :style="{ visibility: getPrevProposalId ? 'visible' : 'hidden' }">
              <i class="material-icons notranslate">chevron_left</i>
            </router-link>
```

But in other places we are using `$router.push` already for this, so I was trying to be consistent